### PR TITLE
JDTB-75 Credentials Encrypt & decrypt to get it

### PR DESCRIPTION
**Pull request by megham2908**
**Ticket:** https://wearejh.atlassian.net/browse/JDTB-75

**Description:**

Created encrypted field in admin to config access key, access id and also cron path.

### CODE REVIEW

File: src/etc/adminhtml/system.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <group id="storage" translate="label"  type="text" sortOrder="10" showInDefault="1" showInStore="0" showInWebsite="0">
+                <label>Storage Details</label>
+                <field id="access_key_id" translate="label" type="obscure" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Access Key Id</label>
+                    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+                </field>
+                <field id="secret_access_key" translate="label" type="obscure" sortOrder="20" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Secret Access Key</label>
+                    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+                </field>
```
Comment (bartoszkubicki) [inline src/etc/adminhtml/system.xml:23]: are all settings now accessible in admin?

Comment (bartoszkubicki) [reply src/etc/adminhtml/system.xml:23]: Cron is good to have - please add all fields into admin config - if we have one, it is logical to have them all. Acl would be nice as well

**Comment by bartoszkubicki** [comment]
@JamelleG could you take a look?

**Review by JamelleG — APPROVED** [review]

**Comment by JamelleG** [comment]
@bartoszkubicki  I have approved from my side, as all looks good.

**Review by bartoszkubicki — APPROVED** [review]

**Pull request merged**